xarray/core/common.py

```diff
@@ -341,14 +341,16 @@ def groupby(self, group, squeeze=True):
         """
         if isinstance(group, basestring):
             group = self[group]
+        elif isinstance(group, (list, tuple)):
+            group = [self[g] if isinstance(g, basestring) else g for g in group]
```

Comment:
Ok, I think I see...here is where the list of groups names is converted to a list of DataArrays.